src/main/kotlin/de/itemis/mps/gradle/modelcheck/Plugin.kt

```diff
@@ -91,6 +91,10 @@ open class ModelcheckMpsProjectPlugin : Plugin<Project> {
                 tasks.create("checkmodels", JavaExec::class.java) {
                     dependsOn(resolveMps)
                     args(args)
+                    if (extension.executable != null) {
+                        executable(extension.executable!!)
+                    }
+
```

Comment:
I see the point. I think doing this change in general is fine. As the MPS execution for the task requires a jvm11 anyway we should also check in case there is no path to the jvm supplied that we are actually running in a jvm11 at the moment. 
```suggestion
                    if (extension.executable != null) {
                          executable(extension.executable!!)
                    } else {
                        if (JavaVersion.current() != JavaVersion.VERSION_11) logger.error("MPS requires Java 11 but current JVM uses ${JavaVersion.current()}, starting MPS will most probably fail!")
                    }
```

Since you changed the `Plugin` data class we should also apply this change to the [Plugin.kt](https://github.com/mbeddr/mps-gradle-plugin/blob/master/src/main/kotlin/de/itemis/mps/gradle/generate/Plugin.kt) of the generate plugin.